src/main/resources/db/migration/V1__create_offender_attendance.sql

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS OFFENDER_ATTENDANCE;
+
+CREATE TABLE OFFENDER_ATTENDANCE
+(
+  ID                              SERIAL PRIMARY KEY,
+  OFFENDER_BOOKING_ID             BIGINT          NOT NULL,
+  EVENT_ID                        BIGINT          NOT NULL,
+  EVENT_DATE                      DATE            NOT NULL,
+  EVENT_LOCATION_ID               BIGINT          NOT NULL,
+  PERIOD                          VARCHAR(2)      NOT NULL,
+  PRISON_ID                       VARCHAR(6)      NOT NULL,
+  ATTENDED                        BOOLEAN,
+  PAID                            BOOLEAN,
+  ABSENT_REASON                   VARCHAR(30),
```

Comment:
WE'll want created by and modified on these fields, keyworker has examples of how these are set.